.github/workflows/update_catalog.yaml

```
@@ -25,16 +25,24 @@ jobs:
       - name: Publish catalog
         run: |
           /bin/bash -c "PWD=${pwd}; /usr/local/bin/apps_catalog_update publish --path $PWD"
+      - name: Check untrack files exist or not
+        id: untrack_files
+        run: |
+          echo ::save-state name=exit_code::$(git -C $PWD --no-pager status --porcelain | wc -l)
```

```suggestion
          echo "exit_code=$(git -C $PWD --no-pager status --porcelain | wc -l)" >> $GITHUB_STATE
```

Should we update it to this?

https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/#examples

Also can't remember if `$GITHUB_STATE` is reflected on the `.outputs`, but `$GITHUB_OUTPUT` should be.